frontend-ssr/services/Utils.ts

```diff
@@ -1 +1,26 @@
 export { v4 as uuid } from 'uuid';
+
+export const timeAgo = (date) => {
+  let ms = (new Date()).getTime() - date.getTime();
+  let seconds = Math.floor(ms / 1000);
+  let minutes = Math.floor(seconds / 60);
+  let hours = Math.floor(minutes / 60);
+  let days = Math.floor(hours / 24);
+
+  if (seconds < 60) {
+    return 'Just now';
+  } if (minutes < 60) {
+    return minutes === 1
+      ? `${minutes} minute ago`
+      : `${minutes} minutes ago`;
+  } if (hours < 24) {
+    return hours === 1
+      ? `${hours} hour ago`
+      : `${hours} hours ago`;
+  } if (days < 24 * 3) {
+    return `${days} days ago`;
+  } if (days > 24 * 3) {
+    return date;
+  }
+  return false;
```

Comment:
Uhm, we should only return strings from this function. Don't quite understand why we need the `false` here.